modules/RTC/RTCBrowserType.js

```diff
@@ -176,14 +176,6 @@ const RTCBrowserType = {
     },
 
     /**
-     * Whether the browser is running on an android device.
-     * @returns {boolean}
```

Comment:
Because it's no longer needed / used. It was there to support a hack which is now gone.